tests/testutils.py

```diff
@@ -75,13 +75,14 @@ def setUp(self):
 
         config = merge_left_with_defaults(CONFIG_DEFAULTS, config)
 
-        self.sygnal = Sygnal(config, reactor)
+        self.sygnal = Sygnal(config, reactor)  # type: ignore
         self.reactor = reactor
 
         start_deferred = ensureDeferred(self.sygnal.make_pushkins_then_start())
 
         while not start_deferred.called:
             # we need to advance until the pushkins have started up
+            assert isinstance(self.sygnal.reactor, ExtendedMemoryReactorClock)
             self.sygnal.reactor.advance(1)
```

Comment:
Since we already have the reactor, maybe use `self.reactor.advance` etc. here and in the two cases below. I think that will let us avoid the assertions.

(Nothing wrong with as it is, just trying to keep things concise where possible)